### ISSUE TITLE
can not access member, member is unknown subplots

### ISSUE THREAD

**Issue by deniserdogan** [microsoft/pyright#3980]
When i write ```_, ax1 = plt.subplots()``` for maplotlib I get the error: ```Cannot access member "pie" for type "ndarray[Any, dtype[Any]]"
  Member "pie" is unknown``` and the same error for ```ax1.axis()```
Everything works fine when I execute code I think something is wrong with the diagnostics.

Here is the whole code:

```python

import matplotlib.pyplot as plt
import numpy as np
import pandas as pd


def passenger_counter_by_class(data: pd.DataFrame) -> None:
    labels = ['1st class', '2nd class', '3rd class']
    sizes = np.array(data.groupby('Pclass').count()['Name'])
    _, ax1 = plt.subplots()
    # 1st error: Cannot access member "pie" for type "ndarray[Any, dtype[Any]]"   Member "pie" is unknown
    ax1.pie(sizes, labels=labels, autopct='%1.1f%%')

    # Second error: Cannot access member "hist" for type "ndarray[Any, dtype[Any]]" Member "hist" is unknown
    ax1.axis('equal')
    plt.show()

```

**_I use pyright on Neovim._**

**Comment by erictraut** [comment]
matplotlib is not a typed library. Do you have type stubs for it? 

If you do not have type stubs, pyright will default to assuming all symbols imported from the matplotlib are `Unknown`, so you should see no errors associated with these types.

I suspect that you have enabled the `useLibraryCodeForTypes` setting, which tells pyright to attempt to infer types from library code even if there are no type annotations. This option is primarily intended for basic completion suggestions, but it is not recommended for use with type checking because inferred types can be inaccurate.

Your options are:
1. Do not enable `useLibraryCodeForTypes`.
2. Find type stubs for matplotlib and install them locally. Partial stubs can be found [here](https://github.com/microsoft/python-type-stubs).
3. Switch to VS Code and Pylance, which ships with partial matplotlib stubs.
